code/src/component/PostList.js

```diff
@@ -0,0 +1,45 @@
+
+/* This Component is used to map json file retuning 
```

Comment:
i like that you have a recap of what the components does. i need to start doing that too!